### PR TITLE
Translate > Getting Started

### PR DESCRIPTION
**Pull request by Darking360**
**Translation of Getting Started page**

**Related:** [Original issue: Spanish Translation Progress](https://github.com/reactjs/es.reactjs.org/issues/4)

**To Do:**

- [x] Finish first translation draft

- [x] Check translation

- [x] Check grammar and orthography

- [x] Submit final

### CODE REVIEW

**Comment by Darking360** [comment]
Greetings! I just finished the translation for the getting started page 🚀 

If any of you could take a look to it, and check if it's ok to be merged would be awesome 💪 

**CC:** @dmoralesm @carburo @alejandronanez

File: content/docs/getting-started.md [hunk 1]
```diff
@@ -18,109 +18,110 @@ redirect_from:
...
+La documentación de React asume cierta familiaridad con la programación en Javascript. No necesitas ser un experto, pero es más difícil aprender React y Javascript al mismo tiempo.
 
->Tip
+Recomendamos pasar por este [resumen de Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para medir tu nivel de conocimiento. Te tomará entre 30 minutos a 1 hora, pero te sentirás más seguro aprendiendo React.
+
+>Consejo
 >
->Whenever you get confused by something in JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) and [javascript.info](http://javascript.info/) are great websites to check. There are also [community support forums](/community/support.html) where you can ask for help.
+>Cuando te sientas confundido sobre algo de Javascript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) y [javascript.info](http://javascript.info/) son sitios increíbles para revisar. También hay un [foro de asistencia por la comunidad](/community/support.html) donde puedes pedir ayuda.
```
Comment (alejandronanez) [inline content/docs/getting-started.md]: Let's link to the Spanish MDN docs!

```suggestion
>Cuando te sientas confundido sobre algo de Javascript, [MDN](https://developer.mozilla.org/es/docs/Web/JavaScript) y [javascript.info](http://javascript.info/) son sitios increíbles para revisar. También hay un [foro de asistencia por la comunidad](/community/support.html) donde puedes pedir ayuda.
```

File: content/docs/getting-started.md [hunk 2]
```diff
@@ -18,109 +18,110 @@ redirect_from:
...
+El [glosario](/docs/glossary.html) contiene un resumen de los términos más comunes que verás en la documentación de React. También hay una sección de preguntas frecuentes dedicada a preguntas cortas y respuestas sobre temas comunes, incluyendo [realizar solicitudes con AJAX](/docs/faq-ajax.html), [estado de un componente](/docs/faq-state.html), y [estructura de archivos](/docs/faq-structure.html).
 
-## Staying Informed
+## Mantente Informado
 
-The [React blog](/blog/) is the official source for the updates from the React team. Anything important, including release notes or deprecation notices, will be posted there first.
+El [Blog de React](/blog/) es la fuente oficial de actualizaciones por parte del equipo de React. Cualquier información importante, incluyendo notas sobre lanzamientos o notas de depreciación, serán puestas allí primero.
 
-You can also follow the [@reactjs account](https://twitter.com/reactjs) on Twitter, but you won't miss anything essential if you only read the blog.
+También puedes seguir la [cuenta @reactjs](https://twitter.com/reactjs) en Twitter, pero no te perderás de nada si solo lees el blog.
```
Comment (alejandronanez) [inline content/docs/getting-started.md]:
```suggestion
También puedes seguir la [cuenta de @reactjs](https://twitter.com/reactjs) en Twitter, pero no te perderás de nada si solo lees el blog.
```

File: content/docs/getting-started.md [hunk 3]
```diff
@@ -18,109 +18,110 @@ redirect_from:
...
+Esta documentación siempre refleja la última versión estable de React, desde React 16, puedes encontrar versiones más antiguas de la documentación en una [página separada](/versions). Recuerda que la documentación para versiones más antiguas es una copia exacta a la del tiempo en que fue lanzada, por lo que no es constantemente actualizada.
 
-## Something Missing?
+## ¿Algo hace falta?
 
-If something is missing in the documentation or if you found some part confusing, please [file an issue for the documentation repository](https://github.com/reactjs/reactjs.org/issues/new) with your suggestions for improvement, or tweet at the [@reactjs account](https://twitter.com/reactjs). We love hearing from you!
+Si algo hace falta en la documentación, o si estas confundido, por favor [abre un caso en el repositorio de la documentación](https://github.com/reactjs/reactjs.org/issues/new) con tus sugerencias para mejoras, o envía un tweet a la [cuenta @reactjs](https://twitter.com/reactjs). ¡Nos encanta saber de ti!
```
Comment (alejandronanez) [inline content/docs/getting-started.md]:
```suggestion
Si algo hace falta en la documentación, o si estas confundido, por favor [abre un caso en el repositorio de la documentación](https://github.com/reactjs/reactjs.org/issues/new) con tus sugerencias para mejoras, o envía un tweet a la [cuenta de @reactjs](https://twitter.com/reactjs). ¡Nos encanta saber de ti!
```

**Comment by Darking360** [comment]
Good morning 🌞 @alejandronanez !

I just committed the requested changes, whenever you can take a look and check if everything is ok would be nice 🙏 

Let me know if anything changes 💪 

**CC:** @carburo @dmoralesm

**Pull request merged**